gallery/quick_start_with_transforms_v2.py

```diff
@@ -0,0 +1,96 @@
+"""
+==================================
+Quick-start with Transforms API v2
+==================================
+
+This example illustrates how to update existing user's data augmentations pipeline with
+new Torchvision Transforms API v2 (:ref:`image transformations <transforms>`).
+
+"""
```

Comment:
Since this gallery doesn't include any code at all, we can put it completely into the top docstring rather than messing around with comments below.